packages/radix-ui-themes/src/components/select.css

```diff
@@ -23,42 +23,38 @@
 .rt-SelectIcon {
   flex-shrink: 0;
 
-  .rt-SelectTrigger:where(:not(.rt-variant-ghost)) & {
+  :where(.rt-SelectTrigger:not(.rt-variant-ghost)) & {
     opacity: 0.9;
   }
 }
 
+/* prettier-ignore */
 .rt-SelectContent {
+  --select-content-radius: min(var(--select-content-corner-height), max(var(--radius-3), var(--radius-full)));
+  --select-content-corner-height: calc(var(--select-item-height) / 2 + var(--select-content-padding));
+  --select-content-scrollbar-margin: max(calc(var(--select-content-radius) / 1.5), var(--select-content-padding));
+  --scrollarea-scrollbar-vertical-margin-top: var(--select-content-scroll-barmargin);
+  --scrollarea-scrollbar-vertical-margin-bottom: var(--select-content-scrollbar-margin);
+  --scrollarea-scrollbar-horizontal-margin-left: var(--select-content-scrollbar-margin);
+  --scrollarea-scrollbar-horizontal-margin-right: var(--select-content-scrollbar-margin);
```

Comment:
I redefined the scrollarea vars on `.radix-themes` so that we can use the custom property cascade when they need to be overridden. No need to target the scrollbar element.